modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -238,7 +238,7 @@ Grant [command]``sudo`` access to the user by adding this line to the [filename]
 <user>  ALL=NOPASSWD: /usr/bin/python, /usr/bin/python2, /usr/bin/python3
 ----
 
-
+This passwordless root access is only needed for onboarding the client. Once the salt-minion is installed, this is no longer needed, because salt-minion is running with root privileges. So it is recommended to remove this line after the minion has been successfully onboarded.
```

Comment:
```suggestion
This procedure grants root access without requiring a password, which is required for registering the client.
When the client is successfully installed it will run with root privileges, so the access is no longer required.
We recommend that you remove the line from the `sudoers` file after the client has been successfully installed.
```

Probably also a good idea to put this in an admonition (note?).